src/components/form-editor/modals/edit-question.tsx

```diff
@@ -144,11 +148,33 @@ const EditQuestion: React.FC<EditQuestionModalProps> = ({ question }) => {
       value: "problem",
     },
   ];
-
+  const check = () => {
```

Comment:
@kumuditha-udayanga What does this helper function accomplish exactly? Can we have a better descriptive naming?